src/itest/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileControllerITest.java

```diff
@@ -51,7 +51,7 @@ void getCompanyProfileWithMatchingCompanyNumber() throws Exception {
         //Updated updated = mock(Updated.class);
```

Comment:
Can this comment and unused variable above be removed?

Reply:
resolved in most recent commit